src/main/java/seedu/address/model/transaction/TransactionContainsPersonNamesPredicate.java

```diff
@@ -12,18 +14,26 @@
  */
 public class TransactionContainsPersonNamesPredicate implements Predicate<Transaction> {
```

Comment:
Need to update the class name since the predicate does not only contain person names now